molecule/amqp_default_interval/converge.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Converge
+  hosts: collectd-test
+  tasks:
+    - include_role:
+        name: '{{ playbook_dir }}/../../../tripleo-collectd-ansible-role'
+      vars:
+        collectd_conf_output_dir: "/etc/collectd.d/"
+        deploy_stage: 'configure_collectd'
+        step: 1
+        # This is not the right things to put here... need to check tripleo_collectd
+        collectd_connection_type: 'amqp1'
+        collectd_amqp_interval: -666
```

Comment:
What is this testing? The rejection of an invalid interval config?

Reply:
Checking whether the default interval is set.
If CollectdAmqpInterval is set to -666, the collectd Interval is used, rather than a plugin-specific interval.
The original behaviour is in the two code snippits below:
 https://github.com/openstack/tripleo-heat-templates/blob/master/deployment/metrics/collectd-container-puppet.yaml#L383-L384
https://github.com/openstack/tripleo-heat-templates/blob/master/deployment/metrics/collectd-container-puppet.yaml#L484-L488